spec/rails_app/app/javascript/application.js

```diff
@@ -0,0 +1,10 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "@hotwired/turbo-rails";
+import { start } from "../../../../src/index";
+import { application } from "./controllers/application";
+import "./controllers";
+
+application.debug = false;
```

Comment:
you have the exact same lines in spec/rails_app/app/javascript/controllers/application.js (and that's where they should be)

Reply:
I have removed them, but just those 2. The `start` call remains in `app/javascript/application.js`. Right?

Also, I did not change those for `.ts` because the typing inside is not clear and it felt irrelevant to do so. In a production application, would you leave it like that or would you make them `.ts`?

Reply:
in a prod application, everything is `.ts`